server/src/main/java/org/apache/uniffle/server/ShuffleServerMetrics.java

```diff
@@ -163,6 +164,7 @@ public class ShuffleServerMetrics {
   public static Gauge.Child gaugeInFlushBufferSize;
   public static Gauge.Child gaugeUsedBufferSize;
   public static Gauge.Child gaugeReadBufferUsedSize;
+  public static Gauge.Child gaugeUsedDirectMemorySize;
```

Comment:
Is this dedicated for netty?

Reply:
For Shuffle Server and monitor Direct memory, but only netty use direct memory 。

Reply:
I think not.

Reply:
I think this is only for netty after reviewing the implementation of `PlatformDependent.usedDirectMemory()`

Reply:
@zuston Ok，I change the class name later , thanks